ecc/enclave_chaincode.go

```diff
@@ -164,11 +169,13 @@ func (t *EnclaveChaincode) init(stub shim.ChaincodeStubInterface) pb.Response {
 	// call enclave
 	responseData, signature, err := t.enclave.Init(jsonArgs, stub, t.tlccStub)
 	if err != nil {
+		logger.Errorf("t.enclave.Init failed. Reason: %s", err)
 		return shim.Error(fmt.Sprintf("ecc: Error while calling CC init enclave: %s", err))
 	}
 
 	enclavePk, err := t.enclave.GetPublicKey()
 	if err != nil {
+		logger.Errorf("init t.enclave.GetPublicKey failed. Reason: %s", err)
```

Comment:
mhh ... if enclave debugging is enabled we also see the following message:
`LOG_ERROR("Lib: ERROR - ecall_get_pk: %d", ret);`